app/models/source_set.rb

```diff
@@ -9,6 +9,7 @@ class SourceSet < ActiveRecord::Base
   validates :name, presence: true
   validates_numericality_of :year, only_integer: true, allow_nil: true,
                                    less_than_or_equal_to: Date.today.year
+  default_scope { order(created_at: :asc) }
```

Comment:
I had to move this out of the [view](https://github.com/dpla/primary-source-sets/pull/99/files#diff-a1380fab38a891ab0e8d364162e964caL2).